bundles/net.tourbook.common/src/net/tourbook/common/MyTourbookNotificationPopup.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (C) 2023 Frédéric Bard
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package net.tourbook.common;
+
+import net.tourbook.common.util.StringUtils;
+
+import org.eclipse.jface.layout.GridDataFactory;
+import org.eclipse.jface.notifications.AbstractNotificationPopup;
+import org.eclipse.jface.resource.ImageDescriptor;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Text;
+
+class MyTourbookNotificationPopup extends AbstractNotificationPopup {
```

Comment:
Can you rename this class to `MTNotificationPopup` like the other `MT...` classes

![image](https://github.com/mytourbook/mytourbook/assets/1283445/6befa0a8-0308-4f49-af13-1be625877e9f)

Reply:
Fixed